data.md

```diff
@@ -158,10 +161,10 @@ The `<op>Word` comparison operators automatically interperet the `Bool` as a `Wo
 ```k
     syntax Word ::= Word "s<Word" Word [function]
  // ---------------------------------------------
-    rule W0:Int s<Word W1:Int => W0 <Word W1                     requires sgn(W0) ==K 1  andBool sgn(W1) ==K 1
-    rule W0:Int s<Word W1:Int => bool2Word(false)                requires sgn(W0) ==K 1  andBool sgn(W1) ==K -1
-    rule W0:Int s<Word W1:Int => bool2Word(true)                 requires sgn(W0) ==K -1 andBool sgn(W1) ==K 1
-    rule W0:Int s<Word W1:Int => twosComp(W1) <Word twosComp(W0) requires sgn(W0) ==K -1 andBool sgn(W1) ==K -1
+    rule W0:Int s<Word W1:Int => W0 <Word W1           requires sgn(W0) ==K 1  andBool sgn(W1) ==K 1
```

Comment:
The sign of a word will always be 1 or -1 right? Shouldn't we switch to ==Int instead?